services/discovery/v1-document-loader.js

```diff
@@ -69,14 +69,18 @@ module.exports = function (RED) {
   function determineSuffix(msg) {
     // Let's assume that if we can't determine the suffix that
     // its a word doc.
-    var ext = '.json',
-      ft = fileType(msg.payload);
+    let ext = '.json';
+    if (! discoveryutils.isJsonObject(msg.payload)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4846228)